src/ansys/units/quantity.py

```diff
@@ -176,41 +145,56 @@ def to(self, to_units: Union[ansunits.Unit, str]) -> "Quantity":
 
         return Quantity(value=new_value, units=to_units)
 
+    def __float__(self):
+        return float(self.value)
+
+    def __array__(self):
+        if "numpy" in sys.modules:
+            return self.value
```

Comment:
This doesn't prove that it's an array though. We could do an `isinstance` check.